src/vg_cluster_mgr.erl

```diff
@@ -118,3 +129,38 @@ head([{_, Host, _, ClientPort} | _]) ->
 
 tail(Nodes) ->
     head(lists:reverse(Nodes)).
+
+ensure_all_topics(ChainName, State) ->
+    Topics = vg_utils:topics_on_disk(),
```

Comment:
This is just for the head node that starts the cluster mgr to load its topics. Other head nodes are going to need to load + add through cluster_mgr that is already running. I made that comment in the chain state module I think.